ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/EnabledGeneratorArguments.cs

```diff
@@ -0,0 +1,11 @@
+// Assembly 'Polly.Core'
+
+using System.Runtime.CompilerServices;
+
+namespace Polly.Simmy;
+
+public readonly struct EnabledGeneratorArguments
```

Comment:
Should this contain a bool so you know if it was turned on or turned off (enabled/disabled)?

Reply:
That's actually the result of the `enabledGenerator`